installer/volcano-development.yaml

```diff
@@ -185,8 +185,8 @@ spec:
       restartPolicy: Never
       containers:
         - name: main
-          image: volcanosh/vc-webhook-manager:latest
-          imagePullPolicy: Always
+          image: volcanosh/vc-webhook-manager:v1.8.0
+          imagePullPolicy: IfNotPresent
```

Comment:
What's the reason to change to `IfNotPresent`

Reply:
Set this parameter to Always in the master branch because the master branch uploads the latest image every day. In this way, the latest image is downloaded each time during installation.

The image of the release-1.8 branch is a fixed version of v1.8.0. Users do not need to download the image every time. Therefore, the IfNotPresent mode is used.

Reply:
It make sense.